django/views/debug.py

```diff
@@ -98,7 +98,7 @@ def get_exception_reporter_filter(request):
 
 class ExceptionReporterFilter(object):
     """
-    Base for all exception reporter filter classes. All overridable hooks
+    Base for all exception reporter filter classes. All overrideable hooks
```

Comment:
I'm happy either way. If you want to use the shorter spelling, I'll change these:
```
django/db/backends/base/schema.py:    # Overrideable SQL templates
django/views/debug.py:    Base for all exception reporter filter classes. All overrideable hooks
tests/test_runner/test_discover_runner.py:    def test_overrideable_test_suite(self):
tests/test_runner/test_discover_runner.py:    def test_overrideable_test_runner(self):
tests/test_runner/test_discover_runner.py:    def test_overrideable_test_loader(self):
```